utils/importer/Importer.js

```diff
@@ -0,0 +1,32 @@
+import fs from 'fs';
+
+class Importer {
+  constructor(models, watcher) {
+    this.models = models;
+    this.import = this.import.bind(this);
+    this.importSync = this.importSync.bind(this);
```

Comment:
If you use arrow functions, it is not necessary to bind methods with a lexical scope. Arrow functions make use of lexical scope by default (https://derickbailey.com/2015/09/28/do-es6-arrow-functions-really-solve-this-in-javascript).